tasks.c

```diff
@@ -4477,8 +4477,17 @@ static void prvResetNextTaskUnblockTime( void )
             uxArraySize = uxTaskGetSystemState( pxTaskStatusArray, uxArraySize, NULL );
 
             /* Create a human readable table from the binary data. */
-            for( x = 0; x < uxArraySize; x++ )
+            for(i = 1; i <= uxArraySize; i++)

```

Comment:
Please add conditional compiling to make this feature optional.  In a large system, this O^2 algorithm could start to take a non-trivial amount of time.